code/modules/mechs/mech_damage.dm

```diff
@@ -99,8 +99,7 @@
 			radiation += damage
 
 	if((damagetype == BRUTE || damagetype == BURN) && prob(25+(damage*2)))
-		sparks.set_up(3,0,src)
-		sparks.start()
+		spark_at(src)
```

Comment:
No `amount`?

Reply:
Again, default is 3, but I can make it explicit if you'd like.

Reply:
Discussed on Discord, needs to either use amount=3 or the other amount=3 should be removed.